### DISCUSSION TITLE
Abundance and occupancy remaining at 0 despite SpDist file with occupant cells

### DISCUSSION THREAD

**Discussion by Tom6161** [RangeShifter/RangeshiftR-tutorials#35] [Q&A] [answered]
Hello,

When I run my simulation (nyears = 100, replicates = 10) for cork oak tree dispersal in Portugal, all my output files have no occupancy or abundance etc. My input SpDist file has 12 cells which I'm using as the initial distribution points however! Even if there was no expansion of the initial population, would the occupancy not be at 10 to start with anyway? Any advice? Happy to post script if necessary.

Best
Tom

**Comment by damariszurell** [comment, marked as answer]
Hi Tom,
This sounds like a mismatch between your habitat map/density dependence parameters and you initial distribution. Check carefully that your "K_or_DensDep" vector in the "ImportedLandscape()" function really matches the number of habitat types ("Nhabitats") and also that the order of K_or_DensDep entries matches the order of the habitat types in your "LandscapeFile". I reckon that you are placing individuals in patches where the "ImportedLandscape()" module expects zero individuals given the habitat types and density dependence parameters.
Cheers,
Damaris
